content/src/Environment.ts

```diff
@@ -393,24 +392,6 @@ export class EnvironmentBuilder {
         'http://localhost:' + env.getConfig<number>(EnvironmentConfig.SERVER_PORT).toString()
     )
 
-    this.registerConfigIfNotAlreadySet(
-      env,
-      EnvironmentConfig.REPOSITORY_QUEUE_MAX_CONCURRENCY,
-      () => process.env.REPOSITORY_QUEUE_MAX_CONCURRENCY ?? RepositoryQueue.DEFAULT_MAX_CONCURRENCY
-    )
-
-    this.registerConfigIfNotAlreadySet(
-      env,
-      EnvironmentConfig.REPOSITORY_QUEUE_MAX_QUEUED,
-      () => process.env.REPOSITORY_QUEUE_MAX_QUEUED ?? RepositoryQueue.DEFAULT_MAX_QUEUED
-    )
-
-    this.registerConfigIfNotAlreadySet(
-      env,
-      EnvironmentConfig.REPOSITORY_QUEUE_TIMEOUT,
-      () => process.env.REPOSITORY_QUEUE_TIMEOUT ?? RepositoryQueue.DEFAULT_TIMEOUT
-    )
-
```

Comment:
Is there an easy way to configure manually those values now? Those vars were used for troubleshooting in the catalyst owner

Reply:
🤔  those values configured the old repository (now deleted). It's still possible to configure query timeouts for the new client tho. would that be enough?

Reply:
yes it is, as long as you can do it through `.env` file right?

Reply:
yes, it's possible with `PG_QUERY_TIMEOUT` and `PG_STREAM_QUERY_TIMEOUT`

Reply:
can we document that in https://github.com/decentraland/catalyst/blob/main/content/README.md ?